Makefile

```diff
@@ -347,8 +355,13 @@ install:all $(PKGCONFBASE)
 	install -m644 libdeflate.h $(DESTDIR)$(INCDIR)
 	install -m755 gzip$(PROG_SUFFIX) \
 		$(DESTDIR)$(BINDIR)/libdeflate-gzip$(PROG_SUFFIX)
-	ln -f $(DESTDIR)$(BINDIR)/libdeflate-gzip$(PROG_SUFFIX)		\
-	      $(DESTDIR)$(BINDIR)/libdeflate-gunzip$(PROG_SUFFIX)
+	if [ -n "$(HARD_LINKS)" ]; then				\
```

Comment:
@ebiggers This works here and should be in the line of your previous comment, I just adjusted it to create a symbolic link instead of a hard copy.

Reply:
Using `$(DESTDIR)$(BINDIR)/libdeflate-gzip$(PROG_SUFFIX)` for the symlink target isn't correct because `$(DESTDIR)` won't be part of the final path.  The two binaries will be in the same directory, so I think you should make it directory-relative: `ln -s libdeflate-gzip$(PROG_SUFFIX) $(DESTDIR)$(BINDIR)/libdeflate-gunzip$(PROG_SUFFIX);`

Reply:
Think this should be OK now? Thanks on the pointers!